lightyear/src/shared/time_manager.rs

```diff
@@ -56,7 +56,7 @@ pub struct TimeManager {
     /// The relative speed set by the client.
     pub base_relative_speed: f32,
     /// Should we speedup or slowdown the simulation to sync the ticks?
-    /// >1.0 = speedup, <1.0 = slowdown
+    /// \>1.0 = speedup, <1.0 = slowdown
```

Comment:
Is this a typo?

Reply:
`/// >` is a table entry just like `/// *` is a bullet point. The rust doc comment has to escape the `>` to denote that the character is meant to represent the greater-than sign and not the start of a table entry. The linter was failing due to that issue.

Reply:
https://rust-lang.github.io/rust-clippy/master/index.html#/doc_lazy_continuation